pkg/model/types.go

```diff
@@ -86,12 +95,25 @@ func NewEndpointFromInstance(inst *types.HttpInstanceSummary) (endpointPtr *Endp
 		return nil, err
 	}
 
+	if endpoint.ServiceType, err = serviceTypeFromAttr(attributes); err != nil {
+		return nil, err
+	}
+
 	// Add the remaining attributes
 	endpoint.Attributes = attributes
 
 	return &endpoint, err
 }
 
+func serviceTypeFromAttr(attributes map[string]string) (serviceType ServiceType, err error) {
```

Comment:
not sure this logic justifies it's own function, duplicated error handling, and temporary variables.

alternatively:
```
if serviceTypeStr, err := removeStringAttr(attributes, ServiceTypeAttr); err != nil {
	return nil, err
}
endpoint.ServiceType = ServiceType(serviceTypeStr)

```